plugins/prefixIds.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+exports.type = 'perItem';
+
+exports.active = false;
+
+exports.params = {
+    delim: '__'
+};
+
+exports.description = 'prefix IDs';
+
+
+var path      = require('path'),
```

Comment:
@caub: webpack should be able to shim or at least ignore fs/path modules: https://github.com/webpack/node-libs-browser